tests/update304.mjs

```diff
@@ -156,7 +156,7 @@ function makeResponse (header, value, lifetime, validatorType, validatorValue) {
   ['X-Frame-Options', 'deny', 'sameorigin'],
   ['X-XSS-Protection', '1', '1; mode=block'],
   ['Cache-Control', 'max-age=1', 'max-age=3600'],
-  ['Expires', utils.httpDate(Date.now(), 1), utils.httpDate(Date.now(), 3600)],
+  ['Expires', 'Tue, 01 Jan 2038 00:00:00 GMT', 'Tue, 19 Jan 2038 03:14:07 GMT'],
```

Comment:
Maybe keep both values well inside of the Y2038 problem, just so it doesn't interfere here?

Reply:
I've changed them (and force-pushed) to `Fri, 01 Jan 2038 01:01:01 GMT` and `Mon, 11 Jan 2038 11:11:11 GMT`; does that seem far enough (>1 week) inside the wrap to you, or what would you suggest?

I'm trying to pick timestamps that are visually quite different so that when they show up in e.g. `results/nginx.json` it's easy to see what happened.

Also, whoops, I got the day-of-week wrong for one of those dates the first time, so I'm glad you made me look at it again.

Reply:
LGTM, thx.